wordpress-mobile/WordPress-iOS
# Remove old Signin code

This removes the old signin code, now unused due to the release of the new login experience.

To test:
- make sure all forms of login still work. Test the really edge cases, especially, stuff like opening a magic link when the app has been killed.

Needs review: @aerych
